awscli/examples/rum/put-rum-events.rst

```diff
@@ -0,0 +1,12 @@
+**To send telemetry events about your application performance and user behavior to CloudWatch RUM**
+
+The following ``put-rum-events`` sends telemetry events about your application performance and user behavior to CloudWatch RUM. If the command succeeds, no output is returned. ::
+
+	aws rum put-rum-events \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general